### DISCUSSION TITLE
[Projects Beta] Cannot filter Pull Requests by `draft:false` (to exclude)

### DISCUSSION THREAD

**Discussion by WizardCM** [community/community#6437] [Projects and Issues]
With a project table of Pull Requests, `is:draft` will show **only** drafts, but there's no way to exclude drafts from a view. Both `draft:false` and `-is:draft` fail to filter results.

**Comment by dgrob** [comment]
Hi @WizardCM the issues says there's one answer, can't see it. Did you find a solution to filter out the drafts?

**Reply by WizardCM** [reply]
There is currently still no way to do this.

**Comment by Shinigami92** [comment]
We are using `status:"Awaiting Review" is:pr -is:draft` and can reproduce this
https://github.com/orgs/faker-js/projects/2/views/9

It's not working :slightly_frowning_face:

**Reply by devshorts** [reply]
`draft:false` seems to work now if anyone is still looking for this

**Comment by joelabreo227** [comment]
`-is:draft` is what I am currently using and appears to be working.
